### ISSUE TITLE
dbus problems. client disconnects

### ISSUE THREAD

**Issue by ManuelSchneid3r** [Rohmilchkaese/spotifyd#1]
This is the best image I found. I simply works. However there is singe issue I encounter every now and then. The client has to connect twice to establish a connection. The logs show some messages about missing dbus then:

````

today at 01:42 Unable to load audio item.
today at 01:42 Os { code: 104, kind: ConnectionReset, message: "Connection reset by peer" }
today at 01:42 /start: line 6: dbus-uuidgen: command not found
today at 01:42 /start: line 7: dbus-daemon: command not found
today at 01:42 Loading config from "/etc/spotifyd.conf"
today at 01:42 No proxy specified
```

**Comment by Rohmilchkaese** [comment]
I‘ll address that one as well. Just give me a few days :) thank for informing me

**Comment by Rohmilchkaese** [comment]
Sorry for not addressing this way too long. I'm happy to confirm I fixed the broken dependency. Also the image is now using the lastest available Spotifyd Version 0.3.3. If you're still using it.
